features/admin.applications.v1/components/forms/share-application-form.tsx

```diff
@@ -216,7 +216,7 @@ export const ApplicationShareForm: FunctionComponent<ApplicationShareFormPropsIn
                 level: AlertLevels.ERROR,
                 message: t(
                     "organizations:notifications." +
-                        "getOrganizationList.genericError.message"
+                        "getOrganizationList.genericError.mupdateEnableIndefiniteAccountLockDurationessage"
```

Comment:
Hope you meant the following :)

```suggestion
                        "getOrganizationList.genericError.updateEnableIndefiniteAccountLockDurationessage"
```

Reply:
Resolved by https://github.com/wso2/identity-apps/pull/6997/commits/6685823eccbff5a04b0ec7abe07b44b2dfbe4910